lib/api/doc.go

```diff
@@ -13,7 +13,21 @@
 // limitations under the License.
 
 /*
-Package api implements the resource struct definitions used by the northbound
-client API.
+Package api implements the struct definitions used on the northbound client API.
+
+In particular this includes:
+	-  The various resource types as exposed on the Northbound API.  The valid resource
+	   types are:
+		-  BGPPeer
+		-  HostEndpoint
+		-  Policy
+		-  Pool
```

Comment:
I'll float that as an option when I put this up for review - and change in a separate PR if agreed.